### ISSUE TITLE
Language Editor 

### ISSUE THREAD

**Issue by hishamco** [OrchardCoreContrib/OrchardCoreContrib.Modules#71]
Adding a feature displaying localizable entries in table that can be manageable, this will simplify the process to add the missing one

**Comment by hyzx86** [comment]
Translation items should be available as a SiteSettings, supporting imports and exports

**Comment by hishamco** [comment]
Translations are not settings, they could be imported and exported, but they require app restart

**Comment by hyzx86** [comment]
That doesn't sound like good news

**Comment by hishamco** [comment]
Settings should be configure something

**Comment by hyzx86** [comment]
Yes maybe we can refresh the translation by reloading the tenant button or directly call the relevant API

**Comment by hyzx86** [comment]
For the design of this feature, please refer to NopCommerce. Its export package feature is very useful for people who don't know how to write code.
Users can update translations for the entire site by editing a list on the interface, and can then export a zip file to share with others.
And for some minority languages, translation packages can be purchased through stores. It's a great design

**Comment by hishamco** [comment]
No code involved here, just you can edit the localization resources from within the UI, perhaps we can import/export them one day

**Comment by giannik** [comment]
Is there an ETA for this module ?

**Comment by hyzx86** [comment]
> No code involved here, just you can edit the localization resources from within the UI, perhaps we can import/export them one day

What I mean by this is to programmatically override the implementation of the translation functionality

**Comment by hishamco** [comment]
@hyzx86 I just finished a simple prototype for this, hope to demo it in the upcoming weeks

**Comment by hyzx86** [comment]
> [@hyzx86](https://github.com/hyzx86) I just finished a simple prototype for this, hope to demo it in the upcoming weeks我刚完成了一个简单的原型，希望在接下来的几周内进行演示

@hishamco 
I've been struggling lately with upgrading from OC 1.8.4 to the latest version…
I'm grilling all kinds of AI Agents 😐
